### ISSUE TITLE
Make the Arcane Elevator only consume Vis on use.

### ISSUE THREAD

**Issue by chrisbecke** [Azanor/thaumcraft-suggestions#65]
The Arcane Elevator has been nerfed into oblivion by a combination of its constant Vis draw and the Flux mechanic of TC5.
The constant vis draw means that it is always draining the aura, and a drained aura produces flux. The only way to remove flux from the aura (short of letting it rain out as taint) is to use pure nodes - but they typically span small, and they get used up by the process. This means they produce less vis, so the aura becomes even more depleted by the elevator, ramping up Flux production.
Basically, in their current form, the things are a hazard. 

I for one would like to actually use them, but since installing TC5 all the elevators I intended to use are sitting in chest.
They really don't seem so OP that this level of hazard is warranted.
Please consider re balancing their vis use / changing their mechanic so I (we) can consider using them again.

**Comment by MalkContent** [comment]
afaik it only uses vis when in use already
edit: checked it: only consumes vis on use, doesn't create flux

**Comment by chrisbecke** [comment]
That must have been sneak buffed. Or my testing was flawed. But when I first installed some with an early 5.4 release they were not ... that fun to have.

**Comment by Hyxaru** [comment]
Yes they only use vis when they actualy levitate an entity. You should close this one.

**Comment by chrisbecke** [comment]
Closed as I was being a numpty
